File: Assets/Plugins/PlayPhoneMultiNetUnity/providers/MNWSLoader.cs
using System;

namespace PlayPhone.MultiNet.Providers
{
  public class MNWSLoader {
    internal const long LOADER_ID_INVALID = -1;
    internal long LoaderId = LOADER_ID_INVALID;

    internal MNWSLoader(long loaderId) {
      this.LoaderId = loaderId;
    }

    public void Cancel() {
      MNDirect.GetWSProvider().CancelRequest(LoaderId);
    }
  }
}

File: Assets/Plugins/Android/src/com/playphone/multinet/providers/MNWSLoader.java
//
//  MNWSLoader.java
//  MultiNet client
//
//  Copyright 2012 PlayPhone. All rights reserved.
//

package com.playphone.multinet.providers;

import com.playphone.multinet.core.ws.IMNWSRequest;

public class MNWSLoader
 {
  public void cancel ()
   {
    request.cancel();
   }

  /*package*/ MNWSLoader (IMNWSRequest request)
   {
    this.request = request;
   }

  private IMNWSRequest request;
 }

